src/SFA.DAS.EmployerIncentives.Api/Models/ActivateCollectionCalendarPeriodRequest.cs

```diff
@@ -0,0 +1,10 @@
+﻿
+namespace SFA.DAS.EmployerIncentives.Api.Models
+{
+    public class ActivateCollectionCalendarPeriodRequest
```

Comment:
I think this is just a CollectionPeriodRequest with Number, Year and IsActive.  It doesn't make sense to call it Activate... when the Active field could be false.